easybuild/easyblocks/generic/bundle.py

```diff
@@ -109,7 +111,7 @@ def __init__(self, *args, **kwargs):
             comp_cfg['easyblock'] = None
 
             # reset list of sources/source_urls/checksums
-            comp_cfg['sources'] = comp_cfg['source_urls'] = comp_cfg['checksums'] = []
+            comp_cfg['sources'] = comp_cfg['source_urls'] = comp_cfg['checksums'] = comp_cfg['patches'] = []
```

Comment:
This is a serious bug. It sets all of them to the same list! So if you do `comp_cfg['sources'].append('foo')` it will also append it to the others unless some copy trickery is done